server/mcp_server_cloudmonitor/src/README.md

```diff
@@ -0,0 +1,190 @@
+                    },
+                    "EndTime": {
+                        "description": "Query the end time in the format of RFC3339 or Unix timestamp",
+                        "type": "number"
+                    },
+                    "Namespace": {
+                        "description": "The cloud product namespace to which the metric belongs",
+                        "type": "string"
+                    },
+                    "SubNamespace": {
+                        "description": "The cloud product subnamespace to which the metric belongs",
+                        "type": "string"
+                    },
+                    "MetricName": {
+                        "description": "The Name of metric",
+                        "type": "string"
+                    },
+                    "Period": {
+                        "description": "Aggregation time period",
+                        "type": "string"
+                    },
+                    "Instances": {
+                        "description": "Instances information",
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "type": "object",
+                                "Dimensions": {
+                                    "description": "The specific dimension information under the instance",
+                                    "type": "array",
+                                    "items": {
+                                        "type": "object",
+                                        "properties": {
+                                            "Name": {
+                                                "type": "string"
+                                            },
+                                            "Value": {
+                                                "type": "string"
+                                            }
+                                        },
+                                        "required": [
+                                            "Name",
+                                            "Value"
+                                        ]
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "required": [
+                    "StartTime",
+                    "EndTime",
+                    "Namespace",
+                    "SubNamespace",
+                    "MetricName"
+                ]
+            }
+        },
+        "required": [
+            "workspaceId",
+            "query",
+            "start",
+            "end"
+        ]
+    }
+}
+```
+
+output：
+
+- Return the index data of the query
+
+#### The most easily evoked Prompt example
+
+```
+Query the metric data of a certain subspace under the cloud product
```

Comment:
would you give a meaningful example prompt? e.g. fetch CPU utilization per min of  ecs instance id <xxxx> in past 12hour